java/core/src/main/java/co/worklytics/psoxy/rules/slack/PrebuiltSanitizerRules.java

```diff
@@ -13,6 +13,7 @@
 public class PrebuiltSanitizerRules {
 
     static final Rules SLACK = Rules.builder()
+        .allowedEndpointRegex("^\\/api\\/discovery\\.enterprise\\.info(?:\\?.+)?")
```

Comment:
test it, but I think all the `\\/` can just be `/` ... I don't think `/` needs to be escaped, as it seems to be treated as a regular character unless it appears in very specific contexts

(I was wrong initially to escape it; imho should remove, as just makes the patterns more verbose)